module/witcher.js

```diff
@@ -11,114 +11,30 @@ with the Hand to Hand Table, page 48 of Witcher TRPG Handbook.
 @param {Actor} actor - The actor passed in from actor-sheet.js to have its properties updated
 */
 function updateDerived(actor){
-    if (actor.data.data.customStat === true){
-        return null;
-    }
-    let thisActor = actor;
-    let base = Math.floor((thisActor.data.data.stats.body.current + thisActor.data.data.stats.will.current)/2);
-	let baseMax = Math.floor((thisActor.data.data.stats.body.max + thisActor.data.data.stats.will.max)/2);
-    let currentBody = thisActor.data.data.stats.body.current;
+	if (actor.data.data.customStat === true){
+		return null;
+	}
+	const thisActor = actor;
+	const stats = thisActor.data.data.stats;
+	const base = Math.floor((stats.body.current + stats.will.current)/2);
+	const baseMax = Math.floor((stats.body.max + stats.will.max)/2);
+	const meleeBonus = Math.ceil((stats.body.current-6)/2)*2;
```

Comment:
Nice!

Reply:
It still seems to have a lot of uncessary spacing. Il try to create a config file that we can all use.